cs3/storage/provider/v1beta1/provider_api.proto

```diff
@@ -724,3 +729,42 @@ message CreateHomeResponse {
   // Opaque information.
   cs3.types.v1beta1.Opaque opaque = 2;
 }
+
+message ProvisionStorageRequest {
+  // OPTIONAL.
+  // Opaque information.
+  cs3.types.v1beta1.Opaque opaque = 1;
+  // REQUIRED.
+  // The owner of the storage.
+  cs3.identity.user.v1beta1.UserId owner = 2;
+  // REQUIRED.
+  // The quota for the user.
+  uint64 quota = 3;
```

Comment:
let's redefine this to:
`uint64 quota_max_bytes`
and add also this field
`uint64 quota_max_files`.

The latter is to limit the number of files a user can have, EOS supports that.